Makefile

```diff
@@ -14,6 +14,12 @@ REPO = github.com/operator-framework/operator-sdk
 BUILD_PATH = $(REPO)/commands/operator-sdk
 PKGS = $(shell go list ./... | grep -v /vendor/)
 
+SOURCE_ANSIBLE_IMAGE = quay.io/operator-framework/ansible-operator
```

Comment:
Curious of the reason behind the `SOURCE` prefix?

Reply:
Yeah that's a little confusing. It makes the most sense in the context of the `image/push` targets.
We need `SOURCE_ANSIBLE_IMAGE` for the e2e builds because the Ansible and Helm operator Dockerfiles expect a specific base image.

Now that I've typed that explanation out, it occurs to me that `ANSIBLE_BASE_IMAGE` might be a better name.